app/frontend/styles/components/search_panel.scss

```diff
@@ -1,10 +1,33 @@
+.search-panel-banner {
+  background-color: $govuk-brand-colour;
+  left: 50%;
+  margin-bottom: govuk-spacing(6);
+  margin-left: -50vw;
+  padding-bottom: govuk-spacing(3);
+  padding-top: govuk-spacing(1);
+  position: relative;
+  width: 100vw;
+}
+
+body {
```

Comment:
why the `body` css selector? is `.pages_home` selector on it on not enough?

Reply:
I just copypastad 🍝  the css block for a different controller class to here, but in fact in a subsequent PR I apply zero padding to `.govuk-main-wrapper` everywhere, so I will just do that in this PR instead